### PR TITLE
fix: show ethereum address in input when scanned from QR code

### PR DESCRIPTION
**Pull request by lvndry**
### 📝 Description

When scanning an ethereum QR Code, the address wasn't filled in the address input.
It was due to the fact that the value of the input was not update after the scan was done.

### ❓ Context

- **Impacted projects**: `LLM`
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/LIVE-8790`

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/screens/SendFunds/02-SelectRecipient.tsx [hunk 1]
```diff
@@ -116,23 +128,24 @@ export default function SendSelectRecipient({ navigation, route }: Props) {
     [account, parentAccount, setTransaction, transaction, setValue],
   );
 
-  // FIXME: PROP IS NOT USED. REMOVE ?
-  // const clear = useCallback(() => onChangeText(""), [onChangeText]);
   const [bridgeErr, setBridgeErr] = useState(bridgeError);
   useEffect(() => setBridgeErr(bridgeError), [bridgeError]);
-  invariant(account, "account is needed ");
```
Comment (lvndry) [inline apps/ledger-live-mobile/src/screens/SendFunds/02-SelectRecipient.tsx:123]: duplicate of line 56

File: apps/ledger-live-mobile/ios/ledgerlivemobile.xcodeproj/xcshareddata/xcschemes/ledgerlivemobile.xcscheme [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
    LastUpgradeVersion = "1210"
-   version = "1.3">
+   version = "1.7">
```
Comment (hedi-edelbloute) [inline apps/ledger-live-mobile/ios/ledgerlivemobile.xcodeproj/xcshareddata/xcschemes/ledgerlivemobile.xcscheme:4]: To remove

File: apps/ledger-live-mobile/metro.config.js [hunk 3]
```diff
@@ -4,18 +4,12 @@
  *
```
Comment (hedi-edelbloute) [inline apps/ledger-live-mobile/metro.config.js:4]: To remove

**Review by hedi-edelbloute — APPROVED** [review]
Some modifications were not needed for the fix so maybe validate with live hub but lgtm

**Pull request merged**